tests/hbacrule/test_hbacrule.yml

```diff
@@ -662,6 +662,129 @@
     register: result
     failed_when: result.changed or result.failed
 
+  # Ensure members are empty.
+  - name: Ensure HBAC rule is present with known members
+    ipahbacrule:
+      ipaadmin_password: SomeADMINpassword
+      ipaapi_context: "{{ ipa_context | default(omit) }}"
+      name: hbacrule01
+      host:
+      - "{{ 'testhost03.' + ipaserver_domain }}"
+      - "{{ 'testhost04.' + ipaserver_domain }}"
+      hostgroup: testhostgroup03,testhostgroup04
+      user: testuser03,testuser04
+      group: testgroup03,testgroup04
+      hbacsvc: testhbacsvc03,testhbacsvc04
+      hbacsvcgroup: testhbacsvcgroup03,testhbacsvcgroup04
```

Comment:
It might be good to add a test in the end to make sure that any of these entries are not there anymore after using the empty lists.

Reply:
I added two tests with `check_mode: yes` to verify if the attributes are empty.

I also refactored the playbook test so that the tests for empty members are in its own test playbook, reducing the size of the base test (which is still too big, IMO).